hyperspace/cosmos/src/chain.rs

```diff
@@ -27,14 +30,35 @@ where
 		todo!()
 	}
 
-	async fn estimate_weight(&self, _messages: Vec<Any>) -> Result<u64, Self::Error> {
-		todo!()
+	async fn estimate_weight(&self, messages: Vec<Any>) -> Result<u64, Self::Error> {
```

Comment:
so that you know, it's standard to put the `_` as a prefix, if the argument is not being used.